metabase/metabase
# Fix static embedding missing dark theme on some date widgets

Closes https://github.com/metabase/metabase/issues/51658

### Description

Fix dark theme regression on static embedding after being introduced by #51596 and #51645

### How to verify

1. Check all static embedding snapshots under `.loki/reference/chrome_laptop_embed_PublicOrEmbedded` and see if all snapshots looks okay on dark theme.

### Note
There is [one flaky test](https://github.com/metabase/metabase/actions/runs/12493072042/job/34863088164?pr=51661) (reran 4 times to get it to pass), but I don't think this PR introduces it
- master: [failed 8/20 times](https://github.com/metabase/metabase/actions/runs/12493991325/job/34863099779#step:9:153)
- This PR: [failed 10/20 times](https://github.com/metabase/metabase/actions/runs/12493994904/job/34863106017#step:9:156)

the difference is within the margin of error.

### Demo

See the Loki snapshot changes in the PR diff

#### Before
- ![image](https://github.com/user-attachments/assets/df32b3f8-7121-4f1e-bf5a-3bd4fa17a297)
- ![image](https://github.com/user-attachments/assets/21628001-da31-43db-8616-8fc1128acd29)
- ![image](https://github.com/user-attachments/assets/fa40b91c-386c-4e96-8b19-a27649fb56d8)
- ![image](https://github.com/user-attachments/assets/68f98dc1-33d7-420c-8a20-8907d7c3f0a0)
- ![image](https://github.com/user-attachments/assets/1be35b9c-5bf6-4f3a-a117-aa892bf92e75)
- ![image](https://github.com/user-attachments/assets/6afdd1a4-2015-4279-80ae-23fdbd3dfba0)
- ![image](https://github.com/user-attachments/assets/cf030a92-e46a-424d-8036-35a76fe90210)
- ![image](https://github.com/user-attachments/assets/db217a14-7971-472c-b814-215e8257ab58)
- ![image](https://github.com/user-attachments/assets/aa576088-5729-4d57-954f-967352ec73d5)
- ![image](https://github.com/user-attachments/assets/a42c336a-4373-4f81-827c-1e5b923b715c)
- ![image](https://github.com/user-attachments/assets/6411f15b-f785-4244-869f-30b12d213edf)
- ![image](https://github.com/user-attachments/assets/2a4a65b4-e7cc-459e-a3ee-7baed9d69dc6)


### Checklist

- [x] Tests have been added/updated to cover changes in this PR (Loki)
